src/utils.js

```diff
@@ -353,9 +353,29 @@ export const getFirstKey = (dict) => {
  * @ignore
  */
 export const getTypicalChromeExecutablePath = () => {
+    /**
+     * Return path of Chrome executable by its OS environment variable to deal with non-english language OS.
+     * Taking also in account the old [chrome 380177 issue](https://bugs.chromium.org/p/chromium/issues/detail?id=380177).
+     *
+     * @returns {string}
+     * @ignore
+     */
+    const getWin32Path = () => {
+        let chromeExecutablePath = 'C:\\%ProgramFiles(x86)%\\Google\\Chrome\\Application\\chrome.exe';
```

Comment:
the default should stay as it was, so not `%ProgramFiles(x86)%` but a real directory name

Reply:
Fixed.
Sorry. I was doing the patch on my laptop in my lunch break.